doc/Language/variables.pod6

```diff
@@ -328,7 +327,7 @@ to that block. This is because the placeholder variables are sorted in Unicode
 order. If you have self-declared a parameter using C<$^a> once, you may refer
 to it using only C<$a> thereafter.
 
-Although it is possible to use nearly any valid identifier as a placeholder
+Although it's possible to use nearly any valid identifier as a placeholder
```

Comment:
You can change this too if you want. In my opinion we shouldn't be turning “it is” to "it's” to make the sentence flow better. It is better rewritten to be more specific or not have to do that at all. Just making the distinction. You can change that to “it is” as well.

I think I incorrectly emphasized the wrong thing in that comment.